src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,125 @@
+import _ from 'lodash';
+import classNames from 'classnames';
+import React from 'react';
+
+import { bindClassNames } from '../../util/style-helpers';
+
+const boundClassNames = bindClassNames('Checkbox');
+
+const {
+	bool,
+	func,
+	object,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["controls", "toggles"]}
+ *
+ * This is a checkbox. It's a component that is in one of two particular states
+ * at any given moment. It features a custom visualization of the native
+ * checkbox button control to reflect its current state.
+ *
+ * It uses a hidden native checkbox control under the hood but leverages other
+ * HTML elements to visualize its state.
+ */
+const Checkbox = React.createClass({
+	propTypes: {
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element.
+		 */
+		className: string,
+
+		/**
+		 * Indicates whether the component should appear and act disabled by
+		 * having a "greyed out" palette and ignoring user interactions.
+		 */
+		isDisabled: bool,
+
+		/**
+		 * Indicates that the component is in the "selected" state when true
+		 * and in the "unselected" state when false.
+		 */
+		isSelected: bool,
+
+		/**
+		 * Called when the user clicks on the component or when they press the
+		 * space key while the component is in focus, and only called when the
+		 * component is in the unselected state. The first parameter is true,
+		 * and the second parameter is the synthetic event object generated by
+		 * React.
+		 */
```

Comment:
"The first parameter is true" -- This is only applicable to the `RadioButton` component since that one cannot be turned "off" via user interaction.

The analogous comment in the `Switch` component is:

```
		/**
		 * Called when the user clicks on the component or when they press the
		 * space key while the component is in focus: the first parameter is a
		 * boolean value set to the opposite of the isSelected prop's value
		 * and the second parameter is the synthetic event object generated by
		 * React.
		 */
```